### PR TITLE
allows to set empty html attr

### PR DESCRIPTION
**Pull request by nowackipawel**
**Description**
as for now there was no way to set empty attribute like required or autofocus

Each pull request should address a single issue, and have a meaningful title.

### CODE REVIEW

**Comment by lonnieezell** [comment]
Looks good, but we need some tests added to test that feature please.

**Comment by nowackipawel** [comment]
@lonnieezell : done.

File: tests/system/Helpers/FormHelperTest.php [hunk 1]
```diff
@@ -810,5 +810,60 @@ public function testSetRadioDefault()
...
+		$expected = 'readonly ';
+		$this->assertEquals($expected, parse_form_attributes(['readonly' => true], []));
+	}
+	
+	
+	// ------------------------------------------------------------------------
+	public function testFormParseFormAttributesFalse()
+	{
+		$expected = 'disabled ';
+		$this->assertEquals($expected, parse_form_attributes(['disabled' => false], []));
```
Comment (lonnieezell) [inline tests/system/Helpers/FormHelperTest.php]: Shouldn't a false setting on a value make it so that it does not appear in the attributes list?

Comment (nowackipawel) [reply tests/system/Helpers/FormHelperTest.php]: I thought we have more flexibility outside form helper function if we will treat true as well as false in the same way.
However I know what you are thinking so I am open to changes.

Comment (lonnieezell) [reply tests/system/Helpers/FormHelperTest.php]: That's fair. I can live with that, and it's still valid HTML, so I'm good. Will merge.

**Pull request merged**